.github/workflows/tests.yaml

```
@@ -48,41 +56,51 @@ jobs:
           name: vault-csi-provider-image
           path: ${{ env.TARBALL_FILE }}
 
-  integration-test:
+  latest-vault:
+    name: latest-vault vault:${{ matrix.vault-version }} kind:${{ matrix.k8s-version }}
```

nittiest of nitpicks, maybe omit the latest version so the job name fits in the github UI?

```suggestion
    name: latest-vault kind:${{ matrix.k8s-version }}
```

---

I dropped the job prefix here:  08c4e68 - I think that looks better now in the UI.